dm/tests/_utils/run_tidb_server

```diff
@@ -5,20 +5,31 @@
 
 set -eu
 
+tmp_config="/tmp/dm_test/tidb.toml"
+
 PORT=$1
 PASSWORD=$2
 CONFIG=""
+
+rm $tmp_config || true
+cat >$tmp_config <<EOF
+new_collations_enabled_on_first_bootstrap = true
+
+[experimental]
+enable-new-charset = true
+
+EOF
+
 if [ "$#" -ge 3 ]; then
-	CONFIG=$3
+	cat $3 >>$tmp_config
```

Comment:
is this for open tidb `new_collation_enabled` and then tidb will throw error when there is unsupported `collation` in `create database or table`?

Reply:
see the comment in https://github.com/pingcap/ticdc/pull/3736/commits/b4bfaae13669e260e9965440c672d8e083de0ebc

Reply:
ok, i see. it will open tidb `new_collation_enabled `.